examples/custom_client.py

```diff
@@ -0,0 +1,21 @@
+import os
+from indico import IndicoClient, IndicoConfig
+
+# Will connect to https://app.indico.io
+client = IndicoClient()
+
+# Environment variables override defaults
+os.environ["INDICO_PROTOCOL"] = "http"
```

Comment:
there no http support, https only

Reply:
In the integration tests I create a client for http environments like this:

`IndicoClient(IndicoConfig(verify_ssl=False))`

Reply:
Yeah config should not have that configurable, basically our authentication scheme only works with https in place.

Reply:
except for single-node :D where we use http
but removed